src/dawn/IIR/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+##===------------------------------------------------------------------------------*- CMake -*-===##
+##                          _                      
+##                         | |                     
+##                       __| | __ ___      ___ ___  
+##                      / _` |/ _` \ \ /\ / / '_  | 
+##                     | (_| | (_| |\ V  V /| | | |
+##                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+##
+##
+##  This file is distributed under the MIT License (MIT). 
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+yoda_add_library(
+  NAME DawnIIR
+  SOURCES 
+          Accesses.cpp  
+          Accesses.h
+          BlockStatements.cpp
+          BlockStatements.h
+          Cache.cpp
```

Comment:
how big should the subproject be? does it make sense to have everything in one folder or should we aim for a separation of "tree" and IIR-Utils?

Reply:
not sure either, 
I think what is important is the depencies of the subproject, that we dont end up in circular dependencies. To me at the moment looks fine, not sure, but still IIR depends on Optimizer, we should finalize it later on, and make it totally independent of Optimizer